setup.py

```diff
@@ -464,6 +464,8 @@ def pxd(name):
                    'pandas/_libs/src/period_helper.h',
                    'pandas/_libs/src/datetime.pxd']
 
+np_dtime_strs_srcs = ['pandas/_libs/src/datetime/np_datetime.c',
```

Comment:
these sources are also used else where, just list them for now directly (e.g. like period.pyx and others do it). This doesn't add anything

Reply:
OK.  This will lead to flake8 warnings that I'm happy to ignore.